account/identity.go

```diff
@@ -170,10 +189,19 @@ func (m *GormIdentityRepository) Delete(ctx context.Context, id uuid.UUID) error
 	err := m.db.Delete(&obj, id).Error
 
 	if err != nil {
+		log.LoggerRuntimeContext().WithFields(map[string]interface{}{
+			"id":  id,
```

Comment:
LoggerRuntimeContext() gives more information such as pid, file, location, etc... so pkg won't be missing. I use LoggerRuntimeContext() when logging an Error to print more information.